tests/test_series.py

```diff
@@ -1288,7 +1337,8 @@ def test_series_min_max_sub_axis() -> None:
     sd = s1 / s2
     check(assert_type(sa, pd.Series), pd.Series)
     check(assert_type(ss, pd.Series), pd.Series)
-    check(assert_type(sm, pd.Series), pd.Series)
+    # TODO GH1089 This should not match to Series[int]
+    check(assert_type(sm, pd.Series), pd.Series)  # pyright: ignore
```

Comment:
@Dr-Irv running into a bit of an issue here, it matches to `Series[int]` so not sure where I got the typehinting wrong, any advice you may have is welcome!